I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 36, in script
    if not enterGame:
  File "game/script.rpy", line 36, in <module>
    if not enterGame:
NameError: name 'enterGame' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 36, in script
    if not enterGame:
  File "D:\renpy-8.3.2-sdk\renpy\ast.py", line 1843, in execute
    if renpy.python.py_eval(condition):
  File "D:\renpy-8.3.2-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "D:\renpy-8.3.2-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 36, in <module>
    if not enterGame:
NameError: name 'enterGame' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.3.2.24090902
Greytale 1.0
Sun Nov 17 01:09:09 2024
